### PR TITLE
chore: switch to the `blue` code formatter 💙

### PR DESCRIPTION
**Pull request by CaselIT**
just a test to see how would `blue` work against our code base

### CODE REVIEW

**Comment by kgriffs** [comment]
<3

**Comment by CaselIT** [comment]
@vytas7 shall we proceed ? at worst we can go back to black without much trouble if blue bets abandoned for any reason.

I think we could keel the black configuration in pyproject, so if someones uses black on it noting much happens

**Comment by vytas7** [comment]
I definitely prefer `blue` to `black` too, not sure if it is worth switching now.
Actually, `blue` defaults to 79-char line length, which is also awesome, IMHO :heart: Maybe we could migrate to `blue` first, and then consider reducing the maximum line length to simply use `blue`'s default settings? :smiling_imp:

**Comment by CaselIT** [comment]
I think 88 is a good compromise :)

> not sure if it is worth switching now though.

well avoids having to manually handle `'` vs `"`, so I guess why not?

**Comment by vytas7** [comment]
We ought to follow `blue`'s suit. If it says 79... (bad poker face)

**Comment by CaselIT** [comment]
@kgriffs @vytas7 updated tox + contributing

File: tox.ini [hunk 1]
```diff
@@ -264,9 +264,9 @@ deps = flake8>=3.7.0
        flake8-import-order
 commands = flake8 []
 
-[testenv:black]
-deps = black>=22.1.0
```
Comment (CaselIT) [inline tox.ini:268]: arg!

**Review by kgriffs — APPROVED** [review]

**Review by vytas7 — APPROVED** [review]
I sneaked in some improvements :smiling_imp: 

I think this is ready to rock :blue_heart:

**Comment by vytas7** [comment]
cc @grantjenks @warsaw :100:

**Comment by warsaw** [comment]
So cool to see Falcon adopt blue.  I remember adopting Falcon for Mailman years ago!

**Pull request merged**